src/App.tsx

```diff
@@ -1,27 +1,67 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
+import { Good } from './types/Good';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+enum SelectedBy {
+  All,
+  FiveFirst,
+  RedGoods,
+}
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-    <button type="button" data-cy="all-button">
-      Load all goods
-    </button>
+  const select = useCallback(async (selection: SelectedBy) => {
+    let selected;
 
-    <button type="button" data-cy="first-five-button">
-      Load 5 first goods
-    </button>
+    switch (selection) {
+      case SelectedBy.FiveFirst:
+        selected = await get5First();
+        setGoods(selected);
```

Comment:
don't repeat yourself: for now you repeated this code several times, can we improve it somehow?